packages/config/config/devices/0x5254/zxt-600_1.3.json

```diff
@@ -104,11 +311,62 @@
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 255,
-			"defaultValue": 5,
+			"defaultValue": 0,
 			"unsigned": true,
```

Comment:
Actually, if you remove `"unsigned": true`, this can be represented entirely without options.
Change the range to be -5 to 5 and restore `allowManualEntry` to `true`.
While you're at it, add `"unit": "°C"`

Reply:
Use to be sure, will this work when -5 uses 251(0xFB)  as its value?

Reply:
Yes, -5 will be transmitted as 251

Reply:
Ah cool. Updated according to your suggestion.